hash/snes.xml

```diff
@@ -5249,6 +5326,31 @@ more investigation needed...
 		</part>
 	</software>
 
+	<software name="squirrel">
+		<description>Squirrel (pirate)</description>
+		<year>1999?</year>
+		<publisher>DVS Electronic Co.?</publisher>
+		<part name="cart" interface="snes_cart">
+			<feature name="slot" value="lorom_poke" />
+			<dataarea name="rom" size="0x200000">
+				<rom name="squirrel-4mbit.sfc" size="0x200000" crc="bad1d9b8" sha1="4a88cdfbfefeaf79df9ebfd1974b16a1571dbce1" />
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- Notes: this is temporarily included for reference, until we fix emulation of the good dump -->
+	<software name="squirrelh" cloneof="squirrel">
```

Comment:
The parent does _seem_ to be working. I can't be sure of the emulation quality, and moreover I just guessed a cart type though.

The comment seems to be boilerplate in that section of the softlist. Since there are other bootleg and cracked ROM pairs that also _appear_ to be working, are not flagged as unsupported, and yet have that same comment, I took that as a sign that maybe someone was being conservative and so I should be too.

Reply:
Squirrel does use the lorom_poke cart type, so you can probably consider the original dump working if the other bootlegs with that cart type are also considered working.

IIRC Hercules may use the Soul Blade protection, but I don't know if it'd work as-is with the current implementation. Try it and see, I guess.

Marvel seems to use a variant of lorom_poke, as nocash describes in this thread: https://forums.nesdev.org/viewtopic.php?p=188779#p188779

KOF98 uses a variant of lorom_poke with bank switching, I described it later in the same thread: https://forums.nesdev.org/viewtopic.php?p=202315#p202315

Reply:
> Squirrel does use the lorom_poke cart type, so you can probably consider the original dump working if the other bootlegs with that cart type are also considered working.

All of them load and seem to work. Digimon doesn't however. The SNES slot implementation is simple enough, but maybe there's a bug elsewhere in the driver?

> IIRC Hercules may use the Soul Blade protection, but I don't know if it'd work as-is with the current implementation. Try it and see, I guess.

Thanks! I tried using lorom_sbld and it didn't work. However, the unhacked parent set soulblad wasn't working either. This one did seem to be a minor mask error in the driver. Now both appear to work.

> Marvel seems to use a variant of lorom_poke, as nocash describes in this thread: https://forums.nesdev.org/viewtopic.php?p=188779#p188779
> 
> KOF98 uses a variant of lorom_poke with bank switching, I described it later in the same thread: https://forums.nesdev.org/viewtopic.php?p=202315#p202315

Cool, maybe I'll look into these more and try to get them working at some point.

Reply:
Ok, I pushed changes and a small fix. I don't plan on looking into these bootlegs any more at the moment. Let me know if there is anything additional that needs addressed with this PR.